src/orchestrator/news_orchestrator.py

```
@@ -289,33 +301,39 @@ def process_articles(self, articles: List[Dict]) -> OrchestrationResult:
-                                    enriched_result = verification_result.copy()
-                                    enriched_result["source"] = article.get("source", "unknown")
-                                    enriched_result["source_specific"] = article.get("source_specific", {})
-                                    
-                                    self.slack_notifier.send_verification_report(enriched_result)
-                                    result.articles_notified += 1
-                                except Exception as e:
-                                    error_msg = f"Notification failed for article: {e}"
-                                    result.errors.append(error_msg)
-                                    self.logger.error(error_msg)
-                        else:
+                        # Slack通知（設定に応じて送信）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)